src/components/App/App.js

```diff
@@ -10,9 +10,14 @@ class App extends Component {
     this.props.getPhotos();
   }
   render() {
+    const handleKeyword = (e) => this.props.handleInput(e.target.value);
+    const handleSearch = () => this.props.searchPhotos(this.props.search);
     return (
       <main>
         <h3>Photo Map App</h3>
+        <input onChange={handleKeyword} type="text"/>
+        <button onClick={handleSearch} type="button">Search</button>
+        <h4>{this.props.status === true ? `#${this.props.search}` : ''}</h4>
```

Comment:
Could be simplified to, {this.props.status && `#${this.props.search}`}
Only is this.props.status will the second part of the conditional be evaluated.